src/pyFAI/detectors/_dectris.py

```diff
@@ -438,6 +459,8 @@ def __init__(self, pixel1=172e-6, pixel2=172e-6, max_shape=None, module_size=Non
 
     def __repr__(self):
         txt = f"Detector {self.name}\t PixelSize= {self._pixel1:.3e}, {self._pixel2:.3e} m"
+        if self.orientation>1:
```

Comment:
```suggestion
        if self.orientation > 0:
```

I guess the orientation info should be displayed also for `orientation = 1`.